aviary/interface/methods_for_level2.py

```diff
@@ -381,8 +382,12 @@ def load_inputs(self, aviary_inputs, phase_info=None, engine_builder=None, verbo
         self.ode_args = dict(aviary_options=aviary_inputs,
                              core_subsystems=default_mission_subsystems)
 
+        self.engine_type = aviary_inputs.get_val(Aircraft.Engine.TYPE)
```

Comment:
Engine type is a GASP mass-only variable that is getting made required here and will break all non-GASP mass aircraft.

The GASPEngineType enum has a ton of types that don't do anything at this level (a user will probably get confused why turboprop adds on the prop automatically, but turboshaft or any of the recip engines don't